pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -177,6 +179,37 @@ export class MobyClient implements ContainerEngineClient {
     }
   }
 
+  async getTags(imageName: string, options?: ContainerBasicOptions) {
```

Comment:
Why does the moby client have some log statements in its `getTags` method that aren't in the nerdctl client's `getTags` method?